docs/contracts.rst

```diff
@@ -1015,6 +1030,8 @@ default constructor: ``contructor() public {}``.
 
 A constructor set as ``internal`` causes the contract to be marked as :ref:`abstract <abstract-contract>`.
```

Comment:
We should also mention that the constructor is meant to run explicitly specified initialization code. In addition to that, the compiler always generates code (this cannot be prevented) that initializes all state variables to their default zero value or the value supplied as part of the inline initialization assignments (please look up which term we use there).

After all that is done, it returns the final code of the contract... (transition to the text below).

Reply:
"The final deployment of the code costs additional gas linear in the length of the code..."

Reply:
@chriseth Done! I hope… :)